Content.Server/DeviceNetwork/NetworkUtils.cs

```diff
@@ -12,25 +12,9 @@ public static class NetworkUtils
         public const int WIRED = 1;
         public const int WIRELESS = 2;
 
-        public const string COMMAND = "command";
-        public const string MESSAGE = "message";
-        public const string PING = "ping";
-
         /// <summary>
-        /// Handles responding to pings.
+        /// The key for command names
         /// </summary>
-        public static void PingResponse<T>(T connection, string sender, IReadOnlyDictionary<string, string> payload, string message = "") where T : IDeviceNetworkConnection
-        {
-            if (payload.TryGetValue(COMMAND, out var command) && command == PING)
-            {
-                var response = new Dictionary<string, string>
-                {
-                    {COMMAND, "ping_response"},
-                    {MESSAGE, message}
-                };
-
-                connection.Send(connection.Frequency, sender, response);
-            }
-        }
+        public const string COMMAND = "command";
```

Comment:
can't these all be an enum instead?

```c#
public enum DeviceType
{  
    Private,
    Wired,
    Wireless
}

public enum PacketType
{
    Command,
    Message,
    Ping
}

Reply:
if you don't want to make COMMAND/etc into an enum because NetworkPayload is a <string, object> store then it should be pascalcase rather than capitalized, and probably have the static class renamed

Reply:
I renamed the class to DeviceNetworkConstants and added the DeviceType enum but i called it ConnectionType.